### ISSUE TITLE
Unnecessary 'util' dependency breaks Expo compatibility

### ISSUE THREAD

**Issue by jmate0321** [configcat/common-js#9]
The following line adds a Node.js dependency that could be easily replaced:

https://github.com/configcat/common-js/blob/b587bdf0c7e977fb8c06608629d4c09365e2523c/src/RolloutEvaluator.ts#L5

Removing this import and defining the following function would make ConfigCat compatible with Expo applications:

```
function isUndefined(obj: any): boolean {
    return obj === undefined;
}
```

I've tested ConfigCat with these changes, and the modified version works nicely on Expo.

Please consider making this change. Thank you.

**Comment by z4kn4fein** [comment]
Hey, thanks for the suggestion, we're going to apply this change

**Comment by z4kn4fein** [comment]
Released in [v2.4.2](https://github.com/configcat/common-js/releases/tag/v2.4.2)

**Comment by jmate0321** [comment]
Thanks!
